PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of your changes in the Title above -->
+<!--- See our Contribution Guidelines here:
+          https://github.com/tripal/tripal/blob/7.x-3.x/CONTRIBUTING.md -->
+<!--- If it fixes an open issue, please add the issue link here. -->
+Issue #
+
+## Type(s) of Change(s)
+<!--- What types of changes does your code introduce? 
+         Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] API-specific change (fix or addition to an API function)
+- [ ] Updates documentation (inline or markdown files)
+
+## Description
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
```

Comment:
> `<!--- If it fixes an open issue, please link to the issue here. -->`

This is duplicated (we moved it to the top).

Reply:
thanks, removed